StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1502/SA1502UnitTests.Classes.cs

```diff
@@ -0,0 +1,158 @@
+﻿namespace StyleCop.Analyzers.Test.LayoutRules
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    using StyleCop.Analyzers.LayoutRules;
+    using TestHelper;
+    using Xunit;
+
+    /// <summary>
+    /// Unit tests for the classes part of <see cref="SA1502ElementMustNotBeOnASingleLine"/>.
+    /// </summary>
+    public partial class SA1502UnitTests : CodeFixVerifier
```

Comment:
You could probably merge the class and the struct test files and use theorys instead.